index.html

```diff
@@ -1 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="nav-bar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
```

Comment:
- [ ] Please move the hamburger button(Icon) to a little right to match the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185749335-2b291600-c17f-438b-b7b9-3c908de47f73.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185749343-624ea134-db0f-4699-9a87-200a90961e8e.png)